torch/ao/quantization/experimental/observer.py

```diff
@@ -111,20 +113,39 @@ def _calculate_qparams(
 
         return (self.gamma, quantization_levels, level_indices)
 
+    def float_to_apot(self, x, levels, indices):
```

Comment:
looks like this is quantization from float to apot int value, why are we attaching this to observer? I think it's better to define a standalone function for it